Resources/doc/reference/installation.rst

```diff
@@ -25,7 +25,7 @@ Retrieve the bundle with composer:
 
 .. code-block:: bash
 
-    $ php composer.phar require sonata-project/media-bundle
+    $ composer require sonata-project/media-bundle
```

Comment:
thats right, but for now this is consistent through all our bundles, because i refatored them all, a few months ago...

maybe we could do something like this with dev-kit?
Doing it manually would be a lot of work with nearly no benefit 😞